jina/peapods/peas/container.py

```diff
@@ -188,8 +188,8 @@ def run(
 
     logger = JinaLogger(name, **vars(args))
 
-    cancel = asyncio.Event()
-    fail_to_start = asyncio.Event()
+    cancel = threading.Event()
```

Comment:
why? it does not really matter right?

Reply:
it does, an `asyncio.Event()` will fail if there is no asyncio loop running, which is not guranteed at this point. The `rolling_update` tests were failing due to this.